### PR TITLE
Update and align dependencies

### PR DESCRIPTION
**Pull request by dave2wave**
When analyzing `mvn dependency:list | sort | uniq` I noticed some version conflicts with dependencies. This PR updates several.

### CODE REVIEW

File: mvn-defaults/pom.xml [hunk 1]
```diff
@@ -355,17 +355,17 @@
...
                 <groupId>com.fasterxml.jackson.jaxrs</groupId>
                 <artifactId>jackson-jaxrs-json-provider</artifactId>
-                <version>2.9.8</version>
+                <version>2.18.1</version>
             </dependency>
             <dependency>
                 <groupId>com.sun.xml.bind</groupId>
                 <artifactId>jaxb-core</artifactId>
-                <version>2.3.0.1</version>
+                <version>4.0.5</version>
```
Comment (jshook) [inline mvn-defaults/pom.xml:363]: This seems like a big change. Sometimes these changes are not runtime compatible. Can we verify this with downstream docs somewhere, or ensure it with some additional verification.

Comment (jshook) [reply mvn-defaults/pom.xml:363]: Or even better, remove the dependency chain if it is vestigial.

**Review by jshook — CHANGES_REQUESTED** [review]
One item for research

File: mvn-defaults/pom.xml (hunk 1 above)
Comment (dave2wave) [reply mvn-defaults/pom.xml]: Removed and the build functioned.

**Comment by dave2wave** [comment]
> One item for research

Research completed

**Review by jshook — APPROVED** [review]
Awesome change!

**Pull request merged**